src/client/prerender.js

```diff
@@ -46,9 +46,8 @@ export function loadPrerenderScripts(initialState) {
 
 export function moveHelmetScripts() {
   document.addEventListener('DOMContentLoaded', () => {
-    const helmetScripts = [...document.querySelectorAll('script[data-react-helmet]')];
+    const helmetScripts = [...document.body.querySelectorAll('script[data-react-helmet]')];
     helmetScripts.forEach((script) => document.body.removeChild(script));
-    helmetScripts.forEach((script) => document.head.appendChild(script));
```

Comment:
so think the one risk with this is that there could be small amount time where the scripts are removed and Helmet has not added them back. what are your thought on the following solution - https://github.com/americanexpress/one-app/commit/cdefce96227e7558931c203a153dbc540c33a230
By delaying render until after the scripts have been moved it should negate the race condition.

Reply:
That is an excellent point and I wish that I had thought of it! :)